mariza/styles.css

```diff
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+.header {
+  background-color: #ff8c00;
+  display: inline-block;
+  color: #ffffff;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+}
+h1 {
+  background-color: #ff8c00;
+  font-size: 30px;
+  font-style: regular;
+  font-weight: medium;
```

Comment:
The only keywords you can use for `font-weight` are `normal` and `bold` -- if you want to use the exact font weights, you need to use the number value for medium, which you can find on Google Fonts here:
<img width="238" alt="Screen Shot 2020-04-09 at 9 33 55 PM" src="https://user-images.githubusercontent.com/2653980/78954412-1b57f680-7aaa-11ea-851f-823b8c187472.png">

```suggestion
  font-weight: 500;
```